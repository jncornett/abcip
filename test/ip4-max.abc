d ( stack="eth:ip4:udp" )
a ( ip4:max=1500; udp:len=2048 )
