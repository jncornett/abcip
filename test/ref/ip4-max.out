IP (tos 0x0, ttl 64, id 1, offset 0, flags [+], proto UDP (17), length 1516)
    10.1.2.3.48620 > 10.9.8.7.8: UDP, length 2048
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  05ec 0001 2000 4011 36ed 0a01 0203 0a09  ......@.6.......
	0x0020:  0807 bdec 0008 0808 0904 4c6f 7265 6d20  ..........Lorem.
	0x0030:  6970 7375 6d20 646f 6c6f 7220 7369 7420  ipsum.dolor.sit.
	0x0040:  616d 6574 2c20 636f 6e73 6563 7465 7475  amet,.consectetu
	0x0050:  7220 6164 6970 6973 6369 6e67 2065 6c69  r.adipiscing.eli
	0x0060:  742e 2045 7469 616d 2073 7573 6369 7069  t..Etiam.suscipi
	0x0070:  742c 2071 7561 6d20 6174 2073 6365 6c65  t,.quam.at.scele
	0x0080:  7269 7371 7565 2075 6c74 7269 6369 6573  risque.ultricies
	0x0090:  2c20 6e69 7369 206e 756c 6c61 2066 6572  ,.nisi.nulla.fer
	0x00a0:  6d65 6e74 756d 206f 7263 692c 2061 2069  mentum.orci,.a.i
	0x00b0:  6d70 6572 6469 6574 206c 6163 7573 2065  mperdiet.lacus.e
	0x00c0:  6e69 6d20 6120 6665 6c69 732e 2043 756d  nim.a.felis..Cum
	0x00d0:  2073 6f63 6969 7320 6e61 746f 7175 6520  .sociis.natoque.
	0x00e0:  7065 6e61 7469 6275 7320 6574 206d 6167  penatibus.et.mag
	0x00f0:  6e69 7320 6469 7320 7061 7274 7572 6965  nis.dis.parturie
	0x0100:  6e74 206d 6f6e 7465 732c 206e 6173 6365  nt.montes,.nasce
	0x0110:  7475 7220 7269 6469 6375 6c75 7320 6d75  tur.ridiculus.mu
	0x0120:  732e 204e 756c 6c61 6d20 6461 7069 6275  s..Nullam.dapibu
	0x0130:  7320 7269 7375 7320 696e 2064 6f6c 6f72  s.risus.in.dolor
	0x0140:  2069 6163 756c 6973 2071 7569 7320 6c61  .iaculis.quis.la
	0x0150:  6369 6e69 6120 6573 7420 7065 6c6c 656e  cinia.est.pellen
	0x0160:  7465 7371 7565 2e20 4d61 6563 656e 6173  tesque..Maecenas
	0x0170:  2075 7420 6665 6c69 7320 6573 742c 2073  .ut.felis.est,.s
	0x0180:  6564 2069 6e74 6572 6475 6d20 6c6f 7265  ed.interdum.lore
	0x0190:  6d2e 2043 7261 7320 6175 6374 6f72 2c20  m..Cras.auctor,.
	0x01a0:  6970 7375 6d20 7669 7461 6520 7665 6e65  ipsum.vitae.vene
	0x01b0:  6e61 7469 7320 6661 7563 6962 7573 2c20  natis.faucibus,.
	0x01c0:  6e75 6e63 206d 6173 7361 2066 6572 6d65  nunc.massa.ferme
	0x01d0:  6e74 756d 2061 7567 7565 2c20 7669 7461  ntum.augue,.vita
	0x01e0:  6520 756c 6c61 6d63 6f72 7065 7220 7075  e.ullamcorper.pu
	0x01f0:  7275 7320 6665 6c69 7320 7175 6973 2061  rus.felis.quis.a
	0x0200:  6e74 652e 2046 7573 6365 2066 6575 6769  nte..Fusce.feugi
	0x0210:  6174 2065 6e69 6d20 7574 206e 756e 6320  at.enim.ut.nunc.
	0x0220:  636f 6e73 6563 7465 7475 7220 6964 2075  consectetur.id.u
	0x0230:  6c74 7269 6369 6573 2074 6f72 746f 7220  ltricies.tortor.
	0x0240:  6c6f 626f 7274 6973 2e20 4d61 6563 656e  lobortis..Maecen
	0x0250:  6173 2069 6e20 6f72 6369 206d 6574 7573  as.in.orci.metus
	0x0260:  2c20 6e6f 6e20 696e 7465 7264 756d 206e  ,.non.interdum.n
	0x0270:  6571 7565 2e20 5065 6c6c 656e 7465 7371  eque..Pellentesq
	0x0280:  7565 2076 756c 7075 7461 7465 2061 6e74  ue.vulputate.ant
	0x0290:  6520 696e 2072 6973 7573 2075 6c6c 616d  e.in.risus.ullam
	0x02a0:  636f 7270 6572 2072 7574 7275 6d2e 2020  corper.rutrum...
	0x02b0:  4675 7363 6520 7665 6c20 6461 7069 6275  Fusce.vel.dapibu
	0x02c0:  7320 6c65 6f2e 2049 6e20 7365 6420 6c61  s.leo..In.sed.la
	0x02d0:  6375 7320 6e65 6320 6c61 6375 7320 7469  cus.nec.lacus.ti
	0x02e0:  6e63 6964 756e 7420 696d 7065 7264 6965  ncidunt.imperdie
	0x02f0:  7420 6120 7665 6c20 6d61 676e 612e 2053  t.a.vel.magna..S
	0x0300:  7573 7065 6e64 6973 7365 206c 7563 7475  uspendisse.luctu
	0x0310:  7320 636f 6e67 7565 2069 6d70 6572 6469  s.congue.imperdi
	0x0320:  6574 2e20 446f 6e65 6320 7365 6420 7361  et..Donec.sed.sa
	0x0330:  6769 7474 6973 206c 6f72 656d 2e20 496e  gittis.lorem..In
	0x0340:  2068 6163 2068 6162 6974 6173 7365 2070  .hac.habitasse.p
	0x0350:  6c61 7465 6120 6469 6374 756d 7374 2e20  latea.dictumst..
	0x0360:  446f 6e65 6320 736f 6c6c 6963 6974 7564  Donec.sollicitud
	0x0370:  696e 2061 6e74 6520 6574 2061 6e74 6520  in.ante.et.ante.
	0x0380:  7469 6e63 6964 756e 7420 7574 2070 6f72  tincidunt.ut.por
	0x0390:  7461 2065 6c69 7420 7268 6f6e 6375 732e  ta.elit.rhoncus.
	0x03a0:  2050 6861 7365 6c6c 7573 2072 6973 7573  .Phasellus.risus
	0x03b0:  206e 756e 632c 2066 7269 6e67 696c 6c61  .nunc,.fringilla
	0x03c0:  2061 2074 696e 6369 6475 6e74 2069 642c  .a.tincidunt.id,
	0x03d0:  206d 616c 6573 7561 6461 2061 7420 7361  .malesuada.at.sa
	0x03e0:  7069 656e 2e20 4372 6173 2076 6974 6165  pien..Cras.vitae
	0x03f0:  206d 6574 7573 206a 7573 746f 2c20 6964  .metus.justo,.id
	0x0400:  2061 7563 746f 7220 6d65 7475 732e 2043  .auctor.metus..C
	0x0410:  7572 6162 6974 7572 2073 7573 6369 7069  urabitur.suscipi
	0x0420:  7420 6661 6369 6c69 7369 7320 6665 6c69  t.facilisis.feli
	0x0430:  7320 7669 7461 6520 7068 6172 6574 7261  s.vitae.pharetra
	0x0440:  2e20 5574 2065 7520 6e69 7369 206e 6973  ..Ut.eu.nisi.nis
	0x0450:  692e 2043 7261 7320 636f 6e64 696d 656e  i..Cras.condimen
	0x0460:  7475 6d20 6575 6973 6d6f 6420 6d61 7373  tum.euismod.mass
	0x0470:  6120 6575 2074 7269 7374 6971 7565 2e20  a.eu.tristique..
	0x0480:  446f 6e65 6320 6572 6f73 206e 6973 6c2c  Donec.eros.nisl,
	0x0490:  2073 656d 7065 7220 6574 206c 6163 696e  .semper.et.lacin
	0x04a0:  6961 2061 2c20 7465 6d70 7573 2069 6420  ia.a,.tempus.id.
	0x04b0:  6469 616d 2e20 4372 6173 206e 6f6e 2074  diam..Cras.non.t
	0x04c0:  656c 6c75 7320 6e6f 6e20 7365 6d20 636f  ellus.non.sem.co
	0x04d0:  6e67 7565 206d 6f6c 6c69 7320 6e6f 6e20  ngue.mollis.non.
	0x04e0:  7574 206d 692e 2044 7569 7320 6469 616d  ut.mi..Duis.diam
	0x04f0:  2066 656c 6973 2c20 6375 7273 7573 206e  .felis,.cursus.n
	0x0500:  6f6e 2073 7573 6369 7069 7420 6e6f 6e2c  on.suscipit.non,
	0x0510:  206d 6f6c 6573 7469 6520 7669 7461 6520  .molestie.vitae.
	0x0520:  7465 6c6c 7573 2e20 446f 6e65 6320 6661  tellus..Donec.fa
	0x0530:  6369 6c69 7369 7320 6475 6920 6574 2065  cilisis.dui.et.e
	0x0540:  726f 7320 706f 7274 7469 746f 7220 636f  ros.porttitor.co
	0x0550:  6e73 6571 7561 742e 2020 5574 2061 2062  nsequat...Ut.a.b
	0x0560:  6962 656e 6475 6d20 6c69 6775 6c61 2e20  ibendum.ligula..
	0x0570:  5072 6f69 6e20 6575 6973 6d6f 6420 706f  Proin.euismod.po
	0x0580:  7274 6120 6c65 6f20 6120 6469 6374 756d  rta.leo.a.dictum
	0x0590:  2e20 4165 6e65 616e 2071 7569 7320 6970  ..Aenean.quis.ip
	0x05a0:  7375 6d20 6d61 7572 6973 2c20 6575 2073  sum.mauris,.eu.s
	0x05b0:  6167 6974 7469 7320 7361 7069 656e 2e20  agittis.sapien..
	0x05c0:  4165 6e65 616e 206e 6962 6820 6d65 7475  Aenean.nibh.metu
	0x05d0:  732c 206c 6f62 6f72 7469 7320 7365 6420  s,.lobortis.sed.
	0x05e0:  6665 7567 6961 7420 7574 2c20 6665 726d  feugiat.ut,.ferm
	0x05f0:  656e 7475 6d20 7669 7461                 entum.vita
IP (tos 0x0, ttl 64, id 1, offset 1496, flags [none], proto UDP (17), length 580)
    10.1.2.3 > 10.9.8.7: udp
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0244 0001 00bb 4011 59da 0a01 0203 0a09  .D....@.Y.......
	0x0020:  0807 6520 6e69 6268 2e20 4372 6173 2073  ..e.nibh..Cras.s
	0x0030:  6365 6c65 7269 7371 7565 2c20 6c69 6775  celerisque,.ligu
	0x0040:  6c61 2065 7420 616c 6971 7565 7420 7665  la.et.aliquet.ve
	0x0050:  7374 6962 756c 756d 2c20 6572 6174 206d  stibulum,.erat.m
	0x0060:  6173 7361 2074 656d 7075 7320 6d69 2c20  assa.tempus.mi,.
	0x0070:  6163 2070 6f72 7474 6974 6f72 2074 7572  ac.porttitor.tur
	0x0080:  7069 7320 6d61 676e 6120 7665 6c20 7572  pis.magna.vel.ur
	0x0090:  6e61 2e20 4e75 6e63 2069 6e20 6c69 6265  na..Nunc.in.libe
	0x00a0:  726f 2061 2061 7567 7565 2076 6573 7469  ro.a.augue.vesti
	0x00b0:  6275 6c75 6d20 6163 6375 6d73 616e 2061  bulum.accumsan.a
	0x00c0:  7420 7574 2064 6961 6d2e 2043 7261 7320  t.ut.diam..Cras.
	0x00d0:  6469 676e 6973 7369 6d20 6661 6369 6c69  dignissim.facili
	0x00e0:  7369 7320 7269 7375 732c 2069 6e20 6469  sis.risus,.in.di
	0x00f0:  6374 756d 2066 656c 6973 2075 6c6c 616d  ctum.felis.ullam
	0x0100:  636f 7270 6572 2061 632e 2053 6564 2068  corper.ac..Sed.h
	0x0110:  656e 6472 6572 6974 206c 6163 7573 2076  endrerit.lacus.v
	0x0120:  6974 6165 206d 6175 7269 7320 636f 6e73  itae.mauris.cons
	0x0130:  6571 7561 7420 6e6f 6e20 6c61 6f72 6565  equat.non.laoree
	0x0140:  7420 6d65 7475 7320 636f 6e67 7565 2e20  t.metus.congue..
	0x0150:  4e75 6c6c 6120 6661 6369 6c69 7369 2e20  Nulla.facilisi..
	0x0160:  4e61 6d20 7574 2064 6f6c 6f72 2065 6c69  Nam.ut.dolor.eli
	0x0170:  742c 2065 7420 6772 6176 6964 6120 6e69  t,.et.gravida.ni
	0x0180:  7369 2e20 5068 6173 656c 6c75 7320 6961  si..Phasellus.ia
	0x0190:  6375 6c69 7320 756c 7472 6963 6573 2070  culis.ultrices.p
	0x01a0:  6f72 7461 2e20 2043 756d 2073 6f63 6969  orta...Cum.socii
	0x01b0:  7320 6e61 746f 7175 6520 7065 6e61 7469  s.natoque.penati
	0x01c0:  6275 7320 6574 206d 6167 6e69 7320 6469  bus.et.magnis.di
	0x01d0:  7320 7061 7274 7572 6965 6e74 206d 6f6e  s.parturient.mon
	0x01e0:  7465 732c 206e 6173 6365 7475 7220 7269  tes,.nascetur.ri
	0x01f0:  6469 6375 6c75 7320 6d75 732e 2041 656e  diculus.mus..Aen
	0x0200:  6561 6e20 6175 6775 6520 7475 7270 6973  ean.augue.turpis
	0x0210:  2c20 6d61 6c65 7375 6164 6120 6163 2076  ,.malesuada.ac.v
	0x0220:  6f6c 7574 7061 7420 6574 2c20 616c 6971  olutpat.et,.aliq
	0x0230:  7561 6d20 6120 6d61 7373 612e 2050 726f  uam.a.massa..Pro
	0x0240:  696e 206e 6f6e 206d 6173 7361 206c 6962  in.non.massa.lib
	0x0250:  6572                                     er
